### PR TITLE
Added needed configuration for laravel

### PR DESCRIPTION
**Pull request by achetronic**
On Laravel, we need to touch a config file to use the library. For that, i documented how to do it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,57 @@ monolog:
...
+            'client_name' => '',
+            'auth' => [
+                'basic' => [
+                    env('LOKI_AUTH_BASIC_USER', ''), 
+                    env('LOKI_AUTH_BASIC_PASSWORD', '')
+                ],
+            ],
+        ],
+    ],
+
```
Comment (rrajkomar) [inline README.md:104]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,57 @@ monolog:
...
+LOKI_SYSTEM_NAME=null
+LOKI_LABELS="${APP_NAME}"
+LOKI_CONTEXT_PREFIX="context_"
+LOKI_EXTRA_PREFIX=
+```
+
+These vars can be injected by Kubernetes, Docker or simply by setting them on the .env file
+
+
+
```
Comment (rrajkomar) [inline README.md:123]: Please the unnecessary extra line breaks

**Review by rrajkomar — CHANGES_REQUESTED** [review]
I'm not familiar with Laravel configuration syntax so I'll rely on you for this.
Please fix remove the unnecessary line breaks as outlined here.
Other than that LGTM.

**Comment by achetronic** [comment]
I will do a pull request without extra spaces and newlines :)

**Pull request closed without merging**